core/src/autogluon/core/scheduler/hyperband.py

```diff
@@ -341,29 +401,43 @@ def add_job(self, task, **kwargs):
                 'bracket': kwargs['bracket'],
                 'reported_result': None,
                 'keep_case': False}
-            milestones = self.terminator.on_task_add(task, **kwargs)
+            first_milestone = self.terminator.on_task_add(task, **kwargs)[-1]
+        # Register pending evaluation(s) with searcher
+        debug_log = self.searcher.debug_log
```

Comment:
is debug_log a standard flag in searcher?

Reply:
Yes, it is defined in BaseSearcher, but returns None per default.